package.json

```diff
@@ -84,6 +85,7 @@
     "@patternfly/react-icons": "^3.14.28",
     "@patternfly/react-styles": "^3.6.15",
     "react-router-dom": "^5.0.1",
-    "react-router-last-location": "^2.0.1"
+    "react-router-last-location": "^2.0.1",
+    "express": "^4.13.4"
```

Comment:
I think express can be a dev dependency?

Reply:
I don't think so - I think only project dependencies are captured with the image creation on openshift.  That's why I put it there or it would fail to start on the move over.